@here/harp-mapview/lib/FrustumIntersection.ts

```diff
@@ -230,39 +228,109 @@ export class FrustumIntersection {
 
     // Computes the rough screen area of the supplied box.
     // TileBounds must be in world space.
-    private computeSubTileArea(tileBounds: THREE.Box3) {
-        if (
-            (!this.m_extendedFrustumCulling ||
-                this.m_mapTileCuller.frustumIntersectsTileBox(tileBounds)) &&
-            this.m_frustum.intersectsBox(tileBounds)
-        ) {
-            const contour = [
-                new THREE.Vector3(tileBounds.min.x, tileBounds.min.y, 0).applyMatrix4(
-                    this.m_viewProjectionMatrix
+    private computeSubTileArea(tileBounds: THREE.Box3 | OrientedBox3) {
```

Comment:
This change was necessary so that when the max visible tile limit is hit, tiles that are less visible(are smaller on screen) are discarded first. But there are still problems at the pole region.

Reply:
Of course I understand the need for it, we will probably improve it anyway when implementing mixed LOD.
Anyway both methods are far from being accurate because only small portion of projected polygon may be actually visible. Imagine situation then only the corner vertex of entire tile passes the screen clipping (the rest is beyond frustum), then this tile is hardly visible (most probably just below camera) but contributes to loading a lot (gets highest priority because it is close and big after projection).
Thus we need to sacrifice here anyway, before finding some better solution. The question is if it will not affect our performance too much - need to be tested.
One minor change I would suggest is to split this method into two, they are used separately anyway and gets clearer interface (for ABB, OBB).